Documentation/grpc-metadata.md

```diff
@@ -224,7 +231,23 @@ func (s *server) SomeStreamingRPC(stream pb.Service_SomeStreamingRPCServer) erro
 }
 ```
 
+Do not use
+[FromOutgoingContext][4] on the server to write metadata to be sent to the client.
+[FromOutgoingContext][4] is for client-side use only.
+
 ## Updating metadata from a server interceptor
 
 An example for updating metadata from a server interceptor is
 available [here](../examples/features/metadata_interceptor/server/main.go).
+
+[1]: <https://pkg.go.dev/google.golang.org/grpc/metadata#FromIncomingContext> "FromIncomingContext"
+[2]: <https://godoc.org/google.golang.org/grpc#SendHeader> "SendHeader"
+[3]: https://godoc.org/google.golang.org/grpc#SetTrailer "SetTrailer"
+[4]: https://pkg.go.dev/google.golang.org/grpc/metadata#FromOutgoingContext "FromOutgoingContext"
+[5]: https://godoc.org/google.golang.org/grpc#ServerStream "ServerStream"
+[6]: https://godoc.org/google.golang.org/grpc "grpc"
+[7]: https://godoc.org/google.golang.org/grpc#ClientStream "ClientStream"
+[8]: https://godoc.org/google.golang.org/grpc#Header "Header"
+[9]: https://godoc.org/google.golang.org/grpc#Trailer "Trailer"
+[10]: https://godoc.org/google.golang.org/grpc#CallOption, "CallOption"
+[11]: https://godoc.org/google.golang.org/grpc/metadata "Metadata"
```

Comment:
Please assign meaningful names instead of numbers (and I think the actual name of the API would work best). With the latter, if entries are added or removed from the list in the future, we might get to a state where the numbers are not contiguous, and that would not look great.

Reply:
Done